scipy/stats/biasedurn.pyx

```diff
@@ -0,0 +1,77 @@
+# distutils: language = c++
+
+from .biasedurn cimport CFishersNCHypergeometric, StochasticLib3
+import numpy as np
+from numpy.random cimport bitgen_t
+# from numpy.random.mtrand cimport RandomState
+from cpython.pycapsule cimport PyCapsule_GetPointer, PyCapsule_IsValid
+
+cdef class _PyFishersNCHypergeometric:
+    cdef CFishersNCHypergeometric c_fnch
+
+    def __cinit__(self, int n, int m, int N, double odds, double accuracy):
+        self.c_fnch = CFishersNCHypergeometric(n, m, N, odds, accuracy)
+
+    def mode(self):
+        return self.c_fnch.mode()
+
+    def mean(self):
+        return self.c_fnch.mean()
+
+    def variance(self):
+        return self.c_fnch.variance()
+
+    def probability(self, int x):
+        return self.c_fnch.probability(x)
+
+    def moments(self):
+        cdef double mean, var
+        self.c_fnch.moments(&mean, &var)
+        return mean, var
+
+
+cdef bitgen_t* make_rng(random_state=None):
```

Comment:
This is where the bitgen_t object is extracted as a default, from the seed, or user provided RandomState or Generator